design/workitemtype.go

```diff
@@ -0,0 +1,178 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+// fieldType is the datatype of a single field in a work item type
+var fieldType = a.Type("fieldType", func() {
+	a.Description("A fieldType describes the values a particular field can hold")
+	a.Attribute("kind", d.String, "The constant indicating the kind of type, for example 'string' or 'enum' or 'instant'")
+	a.Attribute("componentType", d.String, "The kind of type of the individual elements for a list type. Required for list types. Must be a simple type, not  enum or list")
+	a.Attribute("baseType", d.String, "The kind of type of the enumeration values for an enum type. Required for enum types. Must be a simple type, not  enum or list")
+	a.Attribute("values", a.ArrayOf(d.Any), "The possible values for an enum type. The values must be of a type convertible to the base type")
+
+	a.Required("kind")
+})
+
+// fieldDefinition defines the possible values for a field in a work item type
+var fieldDefinition = a.Type("fieldDefinition", func() {
+	a.Description("A fieldDescription aggregates a fieldType and additional field metadata")
+	a.Attribute("required", d.Boolean)
+	a.Attribute("type", fieldType)
+
+	a.Required("required")
+	a.Required("type")
+
+	//	a.View("default", func() {
```

Comment:
I don't know. This is a left-over from moving the code pieces from different locations to a single `design/workitemtype.go` file.